wrappers/src/results_cs.cpp

```diff
@@ -43,6 +44,21 @@ inline T get(Results* results, size_t ndx, NativeException::Marshallable& ex)
     });
 }
 
+inline void alias_backlinks(parser::KeyPathMapping &mapping, const realm::SharedRealm &realm)
```

Comment:
Please use the updated version so that this works across classes, see the fix here: https://github.com/realm/realm-js/pull/1734/files